zephyr/wrapper.c

```diff
@@ -198,8 +193,12 @@ int interrupt_get_irq(unsigned int irq, const char *cascade)
 
 int interrupt_register(uint32_t irq, void(*handler)(void *arg), void *arg)
 {
+#ifdef CONFIG_DYNAMIC_INTERRUPTS
 	return arch_irq_connect_dynamic(irq, 0, (void (*)(const void *))handler,
```

Comment:
Where does this arch_irq_connect_dynamic() goes? I couldn't find it in arch/xtensa, from zephyr project?
Since we are compiling for xtensa I believe it should be there, no?

What am I missing?

Reply:
`int __weak arch_irq_connect_dynamic()` in arch/common/sw_isr_common.c